changelog.md

```diff
@@ -1,6 +1,7 @@
 (Unreleased; add upcoming change notes here)
 ==============================================
 
+* Add support for configuring Athena workgroup
```

Comment:
Could you please add you name at the end? e.g.

`* Add homebrew installation support. ((Thanks: @chenrui333))`